ui/server/src/main/scala/pl/touk/nussknacker/ui/validation/PrettyValidationErrors.scala

```diff
@@ -66,6 +66,7 @@ object PrettyValidationErrors {
       case UnknownProperty(propertyName, _) => unknownProperty(typ, propertyName)
       case InvalidPropertyFixedValue(fieldName, label, value, values, _) => invalidPropertyFixedValue(typ, fieldName, label, value, values)
       case CustomNodeError(_, message, paramName) => NodeValidationError(typ, message, message, paramName, NodeValidationErrorType.SaveAllowed)
+      case CustomServiceValidationError(message, _) => node(message, "")
```

Comment:
I think we also need: CustomParameterValidationError here?

Reply:
No, it is handled as a ParameterValidationError
`case pve: ParameterValidationError => handleParameterValidationError(pve)`

Reply:
Never mind, just removed this class